content/en/docs/setup/install/multicluster/verify/index.md

```diff
@@ -174,3 +174,9 @@ Hello version: v1, instance: helloworld-v1-86f77cd7bd-cpxhv
 
 **Congratulations!** You successfully installed and verified Istio on multiple
 clusters!
+
+## Next Steps
+
+You can now configure
+[locality load balancing](/docs/tasks/traffic-management/locality-load-balancing)
+to control the traffic across your multicluster mesh.
```

Comment:
This isn't quite true. Here you've only got two clusters, but to do the locality load balancing tasks you need 4 clusters with a shared control plane. Maybe something like this instead:
```suggestion
Check out the [locality load balancing tasks](/docs/tasks/traffic-management/locality-load-balancing) 
to learn how to control the traffic across a multicluster mesh.
```